<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventSettings</class>
 <widget class="QWidget" name="EventSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ev√©nement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="le_Name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="cb_Type">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Profil</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sippican</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ponctuel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Archivage Continu</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_Valider">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btn_Annuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
